src/settings.c

```diff
@@ -238,6 +254,22 @@ init_settings_table (Settings   *self,
     }
 }
 
+static void
+fontconfig_changed (FcMonitor *monitor,
+                    Settings *self)
+{
+  const char *namespace = "org.gnome.fontconfig";
+  const char *key = "timestamp";
+  
+  g_debug ("Emitting changed for %s %s", namespace, key);
+
+  self->fontconfig_timestamp = time (NULL);
+
```

Comment:
This is the same that gsd does: use use time()

Reply:
I'm admittedly ignorant where this timestamp is used, but would it be a breaking change to make it monotonic?